backend/oak/output.txt

```diff
@@ -1,13 +0,0 @@
-N3C:4221281 ! Atrial dilatation
```

Comment:
@Sigfried I deleted this file, but just FYI for how I do this sort of thing, I save or move the file to a standard ignored folder. I usually use `_archive`:
<img width="783" alt="Screen Shot 2023-04-27 at 5 01 23 PM" src="https://user-images.githubusercontent.com/13045020/234992511-f1470438-e38c-41a7-9bbf-aca56a24590c.png">

You could also explicitly add the file to the `.gitignore`, which we can do if you feel you need it.